src/server/editorServices.ts

```diff
@@ -1673,6 +1688,18 @@ namespace ts.server {
             this.logger.info(`ConfigFilePresence:: Current Watches: ${watches}:: File: ${configFileName} Currently impacted open files: RootsOfInferredProjects: ${inferredRoots} OtherOpenFiles: ${otherFiles} Status: ${status}`);
         }
 
+        /*@internal*/
+        private logExtendedConfigFileWatchUpdate(extendedConfigFile: NormalizedPath, canonicalConfigFilePath: string, status: ConfigFileWatcherStatus) {

```

Comment:
We do not need this since this watcher is not going to determine and affect file presence for config file.